Issue 12 add videos to readme (#84)
nhs-r-community/NHSRwaitinglist: README.Rmd

## Explanatory Videos

These videos explain more about the Operations Management methodologies which are encapsulated by the package.  Click through to view them on YouTube.

Detailed presentation of method | Recap of method, and presentation of this package
:---:|:---:
[![Thumbnail for link to MAN huddle video](man/figures/NHSRwaitinglist_MAN_huddle_thumbnail.jpg)](https://www.youtube.com/watch?v=NWthhW5Fgls) | [![Thumbnail for link to Health and Care Analytics Conference E-lab video](man/figures/NHSRwaitinglist_HACA_elab_thumbnail.jpg)](https://www.youtube.com/watch?v=3peqTEl_ZAQ)
Midlands Analyst Network Huddle, Nov 2023 | Health and Care Analytics Conference e-lab, Jul 2024

## Installation 

> You can install the current version of {NHSRwaitinglist} from GitHub with:

Reviewer: Could add a reference here to needing a GitHub PAT (e.g. [{usethis} guide](https://usethis.r-lib.org/articles/git-credentials.html#get-a-personal-access-token-pat) or [Happy Git With R](https://happygitwithr.com/https-pat.html?q=personal%20access#https-pat)).

> ```r

Reviewer: ```suggestion
```{r}
```

# install.packages("remotes")
remotes::install_github("nhs-r-community/NHSRwaitinglist", build_vignettes = TRUE)

> 

Reviewer: ```suggestion
```

```

## Getting Started

There is a minimal example below.  To look in more detail at the functions within the package and some more ways of using them, it is a good idea to review the first two vignettes:

1. The [example walkthrough](https://nhs-r-community.github.io/NHSRwaitinglist/articles/example_walkthrough.html) vignette takes a step-by-step walk through the white paper, using the six core functions in this package.  
2. The [three example waiting lists](https://nhs-r-community.github.io/NHSRwaitinglist/articles/three_example_waiting_lists.html) vignette simulates three closely related waiting lists, and uses package functions to explore some of their characteristics.  

> At it's most basic, the package can be used to simulate a waiting list (a dataframe of waiting list addition dates and removal dates), and then compute some important statistics.  Of course, if you already have waiting list data ready to analyse, you can skip the simulation step...

Reviewer: ```suggestion
At its most basic, the package can be used to simulate a waiting list (a dataframe of waiting list addition dates and removal dates), and then compute some important statistics.  Of course, if you already have waiting list data ready to analyse, you can skip the simulation step in the code below.
```

```{r}
# load the package
library(NHSRwaitinglist)

# simulate a waiting list
waiting_list <- wl_simulator(
  start_date = "2020-01-01",
  end_date = "2024-03-31",
  demand = 10, # simulating 10 patient arrivals per week
  capacity = 10.2 # simulating 10.2 patients being treated per week
)

# compute some waiting list statistics
overall_stats <- wl_stats(
  waiting_list = waiting_list,
  target_wait = 18 # standard NHS 18wk target
)

# review the minimal dataset needed to define a waiting list (first 5 rows only)
knitr::kable(head(waiting_list, 5))

# review the waiting list statistics
knitr::kable(overall_stats)

> 

Reviewer: ```suggestion
```

```

## Contributing

If you want to learn more about this project, please join the discussion at the [NHS-R Community Slack](https://nhsrway.nhsrcommunity.com/community-handbook.html#slack) group and the specific channel #managing-waiting-lists.

Please see our 
[guidance on how to contribute](https://tools.nhsrcommunity.com/contribution.html).

This project is released with a Contributor [Code of Conduct](./CODE_OF_CONDUCT.md). 
By contributing to this project, you agree to abide by its terms.

> The simplest way to contribute is to raise an issue detailing the feature or functionality you would like to see added, or any unexpected behaviour or bugs you have experienced.

Reviewer: ```suggestion
The simplest way to contribute is to [raise an issue](https://github.com/nhs-r-community/NHSRwaitinglist/issues) detailing the feature or functionality you would like to see added, or any unexpected behaviour or bugs you have experienced.
```
